apps/app/src/views/routes/NewOrder/components/SelectPharmacyCard/components/SendToPatient.tsx

```diff
@@ -13,16 +24,33 @@ export const SendToPatient = ({ patient }: any) => {
           <Radio isChecked />
           <VStack align="start" spacing={0} wordBreak="break-all">
             {patient.name ? <Text fontWeight="medium">{patient.name.full}</Text> : null}
-            {patient.phone ? (
-              <Link color="gray.500" href={`tel:${patient.phone}`} isExternal>
-                {formatPhone(patient.phone)}
-              </Link>
-            ) : null}
-            {patient.email ? (
-              <Link color="gray.500" href={`mailto:${patient.email}`} isExternal>
-                {patient.email}
-              </Link>
-            ) : null}
+            {patient?.preferredPharmacies?.length || 0 > 0 ? (
```

Comment:
I think this is supposed to be `(patient?.preferredPharmacies?.length || 0) > 0` or even better `(patient?.preferredPharmacies?.length ?? 0) > 0`

The current thing works, but only because it can be:
```
length || false
undefined || false
```

Reply:
good eye!